pkg/util/atomic/value.go

```diff
@@ -14,47 +14,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package atomic
 
 import (
 	"sync"
-	"sync/atomic"
 )
 
 // TODO(ArtfulCoder)
 // sync/atomic/Value was added in golang 1.4
 // Once support is dropped for go 1.3, this type must be deprecated in favor of sync/atomic/Value.
```

Comment:
I spoke to the gang and it seems like we can drop go 1.3 support now.  I apologize for this, but I think the best refactoring here is to replace the caller with https://golang.org/pkg/sync/atomic/ Value and delete this code.